CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
 * Colors: Darken gray and darkGray so they're AA accessible at smaller sizes (#276)
 * Video: Add a gradient overlay on the control bar (#27)
 * Layer: Layer component is now exported for use and has documentation
+* TextArea: Add a hasError prop
```

Comment:
can we add a `(#280)` here for consistency

Reply:
sure